### PR TITLE
test oss files

### PR DESCRIPTION
**Pull request by zhanghang1989**

### CODE REVIEW

File: mobile_cv/model_zoo/scripts/run_collect_models_det.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```
Comment (newstzpz) [inline mobile_cv/model_zoo/scripts/run_collect_models_det.py:1]: Remove the whole file.

File: mobile_cv/model_zoo/scripts/run_create_model_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
```
Comment (newstzpz) [inline mobile_cv/model_zoo/scripts/run_create_model_benchmark.py:1]: Remove the whole file.

File: mobile_cv/model_zoo/scripts/run_eval_model_zoo_cls.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
```
Comment (newstzpz) [inline mobile_cv/model_zoo/scripts/run_eval_model_zoo_cls.py:1]: Remove the whole file.

File: mobile_cv/model_zoo/tests/test_datasets_classy.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
```
Comment (newstzpz) [inline mobile_cv/model_zoo/tests/test_datasets_classy.py:1]: Remove the whole file.

File: mobile_cv/model_zoo/tests/test_job_queue_db.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
```
Comment (newstzpz) [inline mobile_cv/model_zoo/tests/test_job_queue_db.py:1]: Remove the whole file.

File: mobile_cv/model_zoo/tests/test_model_zoo_classy.py [hunk 6]
```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
```
Comment (newstzpz) [inline mobile_cv/model_zoo/tests/test_model_zoo_classy.py:1]: Remove the whole file.

File: mobile_cv/model_zoo/tests/test_model_zoo_fbnet.py [hunk 7]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
```
Comment (newstzpz) [inline mobile_cv/model_zoo/tests/test_model_zoo_fbnet.py:1]: Remove the whole file.

File: mobile_cv/model_zoo/tests/test_model_zoo_vits.py [hunk 8]
```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```
Comment (newstzpz) [inline mobile_cv/model_zoo/tests/test_model_zoo_vits.py:1]: Remove the whole file.

**Review by newstzpz — CHANGES_REQUESTED** [review]

File: mobile_cv/common/misc/local_cache.py [hunk 9]
```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+import contextlib
+import os
+import tempfile
+import time
+from contextlib import contextmanager
+
+import diskcache
```
[the remark this answers is not in the document]
Comment (zhanghang1989) [reply mobile_cv/common/misc/local_cache.py:9]: Shall we ask people to install https://pypi.org/project/diskcache/ ?

[the remark this answers is not in the document]
Comment (zhanghang1989) [reply mobile_cv/common/misc/local_cache.py:9]: Added to the CI dependencies.

**Comment by zhanghang1989** [comment]
closing PR for deleting this branch. Will recreate one

**Pull request closed without merging**